tests/deployment/test_deploy_cli.py

```diff
@@ -32,12 +33,12 @@ def get_all_scripts(db_conn):
     return list(map(lambda x: x[0], all_scripts))
 
 
-def test_deploy_cli_main(get_params, db_conn, register_language_container):
+def test_deploy_cli_main(db_conn, register_language_container):
     args_list = [
-        "--host", get_params["DB_CONNECTION_HOST"],
-        "--port", get_params["DB_CONNECTION_PORT"],
-        "--user", get_params["DB_CONNECTION_USER"],
-        "--pass", get_params["DB_CONNECTION_PASS"],
+        "--host", db_params.host,
```

Comment:
In-database tests are skipped if aws credentials are not set. To check for these I need to either use @skipif or check the aws params inside the test method. The latter option is not a good approach because i am building lanuge_container unnecessarily if aws credentials are not present. For the first option, I need to get the aws parameters before test run. I created aws_params namedtuple for this purpose. I have converted all the necessary params to namedtuple to avoid the confusion of using some of its parameters as fixtures and some as named groups.

Reply:
Maybe add a comment to the code with this reasoning